teamcity/common_build_feature.go

```diff
@@ -0,0 +1,105 @@
+package teamcity
+
+import (
+	"encoding/json"
+)
+
+type CommonBuildFeature struct {
+	id          string
+	featureType string
+	buildTypeID string
+	vcsRootID   string
```

Comment:
Can this field be removed? It's not clear where it's used, or why it can't be extracted from `properties` on an as-needed basis.